howto-gallery/scripts/PistonActivateEvent.ts

```diff
@@ -3,60 +3,7 @@ import * as mc from "@minecraft/server";
 const overworld = mc.world.getDimension("overworld");
 
 /**
- * A simple piston before activate event
- * @param {(message: string, status?: number) => void} log: Logger function. If status is positive, test is a success. If status is negative, test is a failure.
- * @param {mc.Location} location Location to center this sample code around.
- * @see https://learn.microsoft.com/minecraft/creator/scriptapi/minecraft/server/pistonactivatebeforeeventsignal#subscribe
- * @see https://learn.microsoft.com/minecraft/creator/scriptapi/minecraft/server/pistonactivatebeforeevent
- */
-export function pistonBeforeEvent(log: (message: string, status?: number) => void, targetLocation: mc.Vector3) {
```

Comment:
intentional to remove?

Reply:
Yes, piston before event was deprecated. It is removed in the latest versions.